src/proj.h

```diff
@@ -116,20 +205,17 @@ typedef struct { double lam, phi,  z; }  LPZ;
 typedef struct { double d, m,  s; }  PJ_DMS;
 
 /* Geoid undulation (N) and deflections of the vertical (z, e) */
-typedef struct { double   N,   z,  e; }  PJ_NZE;
+typedef struct { double   z,  e, N; }  PJ_ZEN;
```

Comment:
No, not really - and I have changed it again now to EZN, since eta is the east-west component, zeta the north-south, and N is related to heights, so EZN makes most sense